plugins/openshift-image-registry/src/components/OcirImages/OcirImagesCard.tsx

```diff
@@ -87,7 +87,7 @@ export const OcirImagesCard = ({
           </Typography>
           {imageStream.tags?.length
             ? imageStream.tags.map((tag: string, i: number) => (
-                <Chip size="small" label={tag} key={i} />
+                <Chip size="small" label={tag} key={i} /> // NOSONAR
```

Comment:
can we make use of a tag here over the index?

Reply:
I don't want to touch it because I didn't know if a tag could be listed multiple times. It would be unexpected, and we might want to drop the duplicates then as well... I can do that change.